src/controllers/csi/metadata/sqlite.go

```diff
@@ -3,12 +3,21 @@ package metadata
 import (
 	"context"
 	"database/sql"
+	"strconv"
 	"time"
 
+	dynatracev1beta1 "github.com/Dynatrace/dynatrace-operator/src/api/v1beta1"
 	"github.com/mattn/go-sqlite3"
 	"github.com/pkg/errors"
 )
 
+var (
+	dynakubesAlterStatementMaxFailedMountAttempts = `
+	ALTER TABLE dynakubes
+	ADD COLUMN MaxFailedMountAttempts INT NOT NULL DEFAULT ` + strconv.FormatInt(dynatracev1beta1.DefaultMaxFailedCsiMountAttempts, 10) + ";"
```

Comment:
I hope having the constant a 2nd time here won't cause issues if we update it 😅

Reply:
I tested that, set the default value several different numbers and re-built/deployed  without issue :)